docs/docs/guides/integrate/zitadel-apis/access-zitadel-apis.md

```diff
@@ -0,0 +1,237 @@
+
+[Reference documentation for system API](/docs/apis/introduction#system)
+
+## 1. Create a service user
+
+First you need to create a new service user through the console or ZITADEL APIs.
+
+Via Console:
+
+1. In an organization, navigate to Users > Service Users
+2. Click on **New**
+3. Enter a username and a display name
+4. Click on **Create**
+
+Via APIs:
+
+* [Create User (Machine)](/docs/apis/resources/mgmt/management-service-add-machine-user)
+
+## 2. Grant a Manager role to the service user
+
+ZITADEL Managers are Users who have permission to manage ZITADEL itself.
+There are some different levels for managers.
+
+- **IAM Managers**: This is the highest level. Users with IAM Manager roles are able to manage the whole instance.
+- **Org Managers**: Managers in the Organization Level are able to manage everything within the granted Organization.
+- **Project Mangers**: In this level the user is able to manage a project.
+- **Project Grant Manager**: The project grant manager is for projects, which are granted of another organization.
+
+On each level we have some different Roles. Here you can find more about the different roles: [ZITADEL Manager Roles](/guides/manage/console/managers#roles)
+
+To be able to access the ZITADEL APIs your service user needs permissions to ZITADEL.
+
+1. Go to the detail page of your organization
+2. Click in the top right corner the "+" button
+3. Search for your service user
+4. Give the user the role you need, for the example we choose Org Owner (More about [ZITADEL Permissions](/guides/manage/console/managers))
+
+![Add Org Manager](/img/console_org_manager_add.gif)
+
+## 3. Authenticate service user and request token
+
+Service users can be authenticated using private key JWT, client credentials, or personal access token.
+The [service user authentication](../service-users/authenticate-service-users) can be used to make machine-to-machine requests to any Resource Server (eg, an backend service / API) by requesting a token from the Authorization Server (ZITADEL) and sending the short lived token (access token) in the Header of requests.
+
+This guide covers a specific case of service user authentication when requesting access to the [ZITADEL APIs](/docs/apis/introduction).
+While PAT can be used directly to access the ZITADEL APIS, the more secure authentication methods private key JWT and client credentials must include the [reserved scope](/docs/apis/openidoauth/scopes) `urn:zitadel:iam:org:project:id:zitadel:aud` when requesting an access from the token endpoint.
+This scope will add the ZITADEL APIs to the audience of the access token.
+ZITADEL APIs will check if they are in the audience of the access token, and reject the token in case they are not in the audience.
+
+The following sections will explain the more specific authentication to access the ZITADEL APIs.
+
+### Authenticate with private key JWT
```

Comment:
Not sure but do we need this here? As we already have a whole separat section about it?

Couldn't have a look at the preview so far, but feels like its again the same